OpenRA.Mods.Common/Traits/Buildable.cs

```diff
@@ -38,10 +38,13 @@ public class BuildableInfo : TraitInfo<Buildable>
 		[Desc("Sequence of the actor that contains the icon.")]
 		public readonly string Icon = "icon";
 
-		[PaletteReference]
+		[PaletteReference("IsPlayerPalette")]
 		[Desc("Palette used for the production icon.")]
 		public readonly string IconPalette = "chrome";
 
+		[Desc("Custom palette is a player palette BaseName")]
+		public readonly bool IsPlayerPalette = false;
```

Comment:
This doesn't appear to be used anywhere?

Reply:
Sorry, that was stupid. Fixed.